extensions/amp-story/1.0/_locales/en.json

```diff
@@ -218,5 +218,9 @@
   "101": {
     "string": "Shop Now",
     "description": "Label for the shopping CTA button."
+  },
+  "102": {
+    "string": "Featured in this story",
```

Comment:
Any preference between "Featured in this story" vs "Featured on this page"? The latter might be more clear, but I don't feel too strongly here

Reply:
From the discussion with design, "page" could be confusing to the user since it could be based on context. It might not be clear what "page" refers to since that is language that we use in the format but may be unfamiliar to others.

We can certainly change this in the future if there is confusion.